src/storage/compactor/src/lib.rs

```diff
@@ -89,6 +89,15 @@ pub struct CompactorOpts {
     #[clap(long, env = "RW_OBJECT_STORE_READ_TIMEOUT_MS", value_enum)]
     #[override_opts(path = storage.object_store_read_timeout_ms)]
     pub object_store_read_timeout_ms: Option<u64>,
+
+    pub is_shared_compactor: bool,
+    endpoint: String,
```

Comment:
Need a better naming to indicate what endpoint this is. How about just naming it `proxy_endpoint`?

Reply:
Agree, let's call it `proxy_rpc_endpoint`